gpu-operator/release-notes.rst

```diff
@@ -34,6 +34,36 @@ See the :ref:`GPU Operator Component Matrix` for a list of software components a
 
 ----
 
+.. _v24.6.1:
+
+24.6.1
+======
+
+.. _v24.6.1-new-features:
+
+New Features
+------------
+
+* Added support for the following software component versions:
+
+  - NVIDIA Container Toolkit v1.16.2
```

Comment:
We did not bump the toolkit version
```suggestion
```

Reply:
TY.  I meant to add ` NVIDIA GPU Feature Discovery for Kubernetes v0.16.2`.